tests/suites/test_suite_ssl.function

```diff
@@ -52,4 +80,45 @@ void ssl_set_hostname_twice( char *hostname0, char *hostname1 )
 
     mbedtls_ssl_free( &ssl );
 }
-/* END_CASE */
\ No newline at end of file
+/* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_SSL_PREVERIFY_CB:MBEDTLS_FS_IO:MBEDTLS_KEY_EXCHANGE_RSA_ENABLED:MBEDTLS_AES_C:MBEDTLS_SHA256_C:MBEDTLS_CIPHER_MODE_CBC */
```

Comment:
Lots of dependencies! Will this test case execute in every build we need it to?

I think we may need to simplify the test. I don't want it to be unnecessarily skipped if one of the test dependencies is missing.